package.json

```diff
@@ -10,7 +10,7 @@
     "material-ui"
   ],
   "scripts": {
-    "build": "babel -d . src; node ./src/makeTheme.js",
+    "build": "babel -d . src; node ./src/makeTheme.js; cp ./src/styles/animation.css ./styles",
```

Comment:
We might want to consider adding a `./scripts` directory and wrap these last two actions in that.